tests/helpers.py

```diff
@@ -162,3 +162,14 @@ def run_request(client_method, url, data=None):
         )
 
     return _response
+
+
+def run_request_octet(client_method, url, data=None):
+    if data is None:
+        _response = client_method(url, content_type="application/octet-stream")
```

Comment:
I think you can return this directly, without assigning `_response`. Then later, you can remove the "else" keyword and return the other value directly.